js/explore/dependencies/force_dependencyGraph.js

```diff
@@ -605,7 +605,14 @@ function draw_force_graph(areaID, adjacentAreaID) {
                     .attr('r', 5)
                     .attr('class', 'inGraph')
                     .attr('language', '')
-                    .attr('id', d => d.id)
+                    .attr('id', d => {
+                        if (d.id === 'github') {
```

Comment:
I guess I'm not sure what causes the error exactly, but would it be better to do something like `if (d.id.lower() === 'github') {` for this instead? That way you don't have to care anything about the casing of the word "github"

Reply:
The issue come down to the fact that this:
![image](https://user-images.githubusercontent.com/29114346/93502472-9b8d4e00-f8cb-11ea-97f1-8fc1700f082f.png)
Has the id "github"

Thus, if the circle had any ID other than github (all lower case) including something like Github, it would be ok.

Reply:
The GitHub logo button in the header has an id `github`. So having that id in the page content as well could cause problems.

This does seem like an awkward hack though. I would think the real solution here would be to give the header button a more specific/unique id, e.g. `github-icon`. (In fact, the Twitter button is for some reason using the `github` id too, which should be fixed.)

Reply:
Got it, thanks!